assembly/assembly-ide-war/pom.xml

```diff
@@ -463,4 +278,44 @@
             </plugin>
         </plugins>
     </build>
+    <profiles>
```

Comment:
If I understand correctly, to do what you propose I would have to add directly all (but the `dashboard-client`) transitive dependencies of the upstream `assembly-ide-war` project to the main (non-profile) `<dependencies>` list. I don't see how it would make the build easier to read and maintain.